docs/tutorials/istio.md

```diff
@@ -0,0 +1,28 @@
+# Configure Istio Service Mesh
+
+Integrating Istio's service mesh with Coder's Ingress enables powerful traffic management, security, and observability capabilities. By placing Coder's workspace traffic behind Istio's intelligent proxy layer, you can implement access controls, encrypt service-to-service communication, and gain visibility into your workspace network patterns. This guide walks through the process of configuring Istio alongside Coder's existing ingress controller, ensuring that developer workspaces remain accessible while benefiting from Istio's comprehensive service mesh features.
+
+Before proceeding, ensure you have a running Kubernetes cluster with both Coder and Istio installed, and that you have administrative access to configure both systems. Once you have access to your Coder cluster, apply the following manifest:
+
+```yaml
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
I wasn't sure about putting an `alpha` version in our docs so I had to hunt a bit to confirm the version - seems old but Istio's docs still refer to it <https://istio.io/latest/docs/reference/config/networking/envoy-filter/>

Reply:
Yep, this is common in Kubernetes-land. I'm not familiar enough with Istio / Envoy to know if there is a different way to accomplish the same task using `networking.istio.io/v1` APIs.